### PR TITLE
Replace has/have to with must

### PR DESCRIPTION
**Pull request by JoshuaAdemola**

### CODE REVIEW

File: content/concepts/introduction-bundle/api.md [hunk 1]
```diff
@@ -5,10 +5,10 @@ weight: 70
...
 
-In contrast to many other IoT platforms, {{< product-c8y-iot >}} uses the same APIs and the same interface technology for all use cases. As a consequence, you have a wider range of choices in putting intelligence into your IoT devices, depending on how powerful they are. You also have to use only one set of APIs and one technology to build a complete solution from device to application on your own.
+In contrast to many other IoT platforms, {{< product-c8y-iot >}} uses the same APIs and the same interface technology for all use cases. As a consequence, you have a wider range of choices in putting intelligence into your IoT devices, depending on how powerful they are. You also must use only one set of APIs and one technology to build a complete solution from device to application on your own.
```
Comment (BeateRixen) [inline content/concepts/introduction-bundle/api.md:8]: I would change this to "need to", as this is what it means here.

File: content/datahub/getting-started-with-datahub-bundle/accessing-and-logging-in.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ You access {{< product-c8y-iot >}} DataHub via a web browser. It has been tested
 
 ### How to log into DataHub
 
-After the {{< product-c8y-iot >}} DataHub services have been subscribed for your tenant, you have to log into your tenant. In the **application switcher** you will find the DataHub button. When you click this button, you will be taken to the home page of DataHub. DataHub provides the management and monitoring UI of {{< product-c8y-iot >}} DataHub. Alternatively your tenant administrator can give you the URL directly navigating to DataHub, which requires a login as well.
```
Comment (BeateRixen) [inline content/datahub/getting-started-with-datahub-bundle/accessing-and-logging-in.md:16]: Please revert the changes in the datahub files here and below.

File: content/device-sdk/device-sdk-rest-bundle/device-integration.md [hunk 3]
```diff
@@ -475,7 +475,7 @@ Note that all data types in {{< product-c8y-iot >}} can include arbitrary extens
 
 #### Step 11: Send alarms
 
-Alarms represent events that most likely require human intervention to be solved. For example, if the battery in a device runs out of energy, someone has to visit the device to replace the battery. Creating an alarm is technically very similar to creating an event.
+Alarms represent events that most likely require human intervention to be solved. For example, if the battery in a device runs out of energy, someone must visit the device to replace the battery. Creating an alarm is technically very similar to creating an event.
```
Comment (BeateRixen) [inline content/device-sdk/device-sdk-rest-bundle/device-integration.md:478]: Better replace with "needs to"

File: content/device-sdk/mqtt-bundle/device-certificates.md [hunk 4]
```diff
@@ -304,15 +304,15 @@ If you are not using the intermediate certificate then the command is:
...
 3. If you do not have the server certificate, get it by the command: `openssl s_client -showcerts -connect <cumulocity url>:<mqtt mutual ssl port (currently 8883, but that can be changed in the future)> | openssl x509 -outform PEM > serverCertificate.pem`
-4. Now you can create a truststore, which will contain the server certificate. It has to be created with Java Keytool (openssl does not support creating truststore, so if you don't want to use Java keytool then you will have to keep every trusted certificate in a separate pem file).
-   Remember that `alias` is the unique identifier for every keystore or truststore entry. It means that if you want to add a second trusted certificate to the same truststore then you will have to change the alias from `servercertificate` in the command below to some other name:
+4. Now you can create a truststore, which will contain the server certificate. It must be created with Java Keytool (openssl does not support creating truststore, so if you don't want to use Java keytool then you must keep every trusted certificate in a separate pem file).
```
Comment (BeateRixen) [inline content/device-sdk/mqtt-bundle/device-certificates.md:314]: pem file > PEM file (sorry, just came across that :-))

File: content/device-sdk/mqtt-bundle/device-certificates.md [hunk 5]
```diff
@@ -304,15 +304,15 @@ If you are not using the intermediate certificate then the command is:
...
+   Remember that `alias` is the unique identifier for every keystore or truststore entry. It means that if you want to add a second trusted certificate to the same truststore then you will must change the alias from `servercertificate` in the command below to some other name:
```
Comment (BeateRixen) [inline content/device-sdk/mqtt-bundle/device-certificates.md:315]: will must change > must change

File: content/device-sdk/mqtt-bundle/device-certificates.md [hunk 6]
```diff
@@ -332,7 +332,7 @@ Upload your CA (or intermediate) certificate to the platform. This operation wil
 
 **Via UI:**
 
-1. You have to open the device management application, then navigate to the **Management** tab and select **Trusted certificates**.
+1. You must open the device management application, then navigate to the **Management** tab and select **Trusted certificates**.
```
Comment (BeateRixen) [inline content/device-sdk/mqtt-bundle/device-certificates.md:335]: You must open the device management application > Open the Device Management application

File: content/device-sdk/mqtt-examples-bundle/hello-mqtt-nodejs.md [hunk 7]
```diff
@@ -121,7 +121,7 @@ Before running the application, the MQTT middleware must be installed. To achiev
 $ npm install
 ```
 
-Installation has to be done only once. Afterwards, you only need to execute the following command:
+Installation must be done only once. Afterwards, you only need to execute the following command:
```
Comment (BeateRixen) [inline content/device-sdk/mqtt-examples-bundle/hello-mqtt-nodejs.md:124]: must > needs to

File: content/device-tutorials/raspberry-pi-4.md [hunk 8]
```diff
@@ -94,7 +94,7 @@ sudo su -
 
 To do the edit, you can start the "vi" editor in the superuser console and make the changes there. The vi editor is available with the Raspbian delivery.
 
-If you prefer to use another editor, you may have to install it first. After the editor is installed, ensure that you start it from a console with superuser privileges.
+If you prefer to use another editor, you must install it first. After the editor is installed, ensure that you start it from a console with superuser privileges.
```
Comment (BeateRixen) [inline content/device-tutorials/raspberry-pi-4.md:97]: please leave; "may have to" has a different meaning we should keep

File: content/microservice-sdk/java-bundle/developing-microservice.md [hunk 9]
```diff
@@ -617,7 +617,7 @@ A Spring Boot library was upgraded to 2.5.8, hence upgrading Microservice SDK to
     ```
 * Jackson 2.12.x does not provide the Joda Module by default, it might be required to add `jackson-datatype-joda` dependency and define Joda Module:
   `new ObjectMapper().addModule(new JodaModule());` in a custom Microservice code.
-* Spring Boot 2.5.8 does not provide the _Bean Validation 2.0_ provider  as a transitive dependency anymore. Developers may have to explicitly define a validation provider, for example `hibernate-validator`, or add the `spring-boot-starter-validation` dependency.
+* Spring Boot 2.5.8 does not provide the _Bean Validation 2.0_ provider  as a transitive dependency anymore. Developers must explicitly define a validation provider, for example `hibernate-validator`, or add the `spring-boot-starter-validation` dependency.
```
Comment (BeateRixen) [inline content/microservice-sdk/java-bundle/developing-microservice.md:620]: please leave; "may have to" has a different meaning we should keep

File: content/protocol-integration/cloud-fieldbus-bundle/connecting-fieldbus.md [hunk 10]
```diff
@@ -27,7 +27,7 @@ To connect a Modbus/RTU device, follow these steps:
 2. Enter a name for the device and select the device protocol from the dropdown field. See [Configuring fieldbus device protocols](#configuring-fieldbus) for information on how to add a new device protocol. Set the Modbus address of the connected device.
 3. Click **Add**. {{< product-c8y-iot >}} will now send a notification to the Modbus terminal that a new device is ready to be managed. This may take a few seconds.
 
-After completion, a new child device has been added to the terminal and can now be managed. You can click on the name of the device in the list to navigate to the device. If you have not yet added Modbus devices to the terminal, you may have to reload your browser window to make the **Child Devices** tab visible.
+After completion, a new child device has been added to the terminal and can now be managed. You can click on the name of the device in the list to navigate to the device. If you have not yet added Modbus devices to the terminal, you must reload your browser window to make the **Child Devices** tab visible.
```
Comment (BeateRixen) [inline content/protocol-integration/cloud-fieldbus-bundle/connecting-fieldbus.md:30]: please leave; "may have to" has a different meaning we should keep

File: content/protocol-integration/cloud-fieldbus-bundle/connecting-fieldbus.md [hunk 11]
```diff
@@ -73,7 +73,7 @@ To connect a CAN device, follow these steps:
 
 {{< product-c8y-iot >}} will now send a notification to the fieldbus terminal that a new device is ready to be managed. This may take a few seconds.
 
-After completion, a new child device has been added to the terminal and can now be managed. You can click on the name of the device in the list to navigate to the device. If you have not yet added fieldbus devices to the terminal, you may have to reload your browser window to make the **Child devices** tab visible.
+After completion, a new child device has been added to the terminal and can now be managed. You can click on the name of the device in the list to navigate to the device. If you have not yet added fieldbus devices to the terminal, you must reload your browser window to make the **Child devices** tab visible.
```
Comment (BeateRixen) [inline content/protocol-integration/cloud-fieldbus-bundle/connecting-fieldbus.md:76]: please leave; "may have to" has a different meaning we should keep

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md [hunk 12]
```diff
@@ -37,13 +37,13 @@ After creating an "Associated user" with the proper group and profiles navigate
...
 
-The following information has to be provided:
+The following information must be provided:
 
 - **Login:** The login token is located in the API access entry in the Sigfox Cloud.
 - **Password:** The password token is located in the API access entry in the Sigfox Cloud next to **Password**.
 - **Parent Group ID:** This ID is written in your URL when you are logged into your Sigfox account and you have selected the "Cumulocity" group. For example, "https://backend.sigfox.com/group/**9823ruj29j9d2j9828hd8**/info".
 
-> **Info:** The group name in the screenshot below is only an example. It does  not necessarily have to be "Cumulocity".
+> **Info:** The group name in the screenshot below is only an example. It must not necessarily be "Cumulocity".
```
Comment (BeateRixen) [inline content/protocol-integration/sigfox-bundle/connectivity-sigfox.md:46]: Please leave to keep the meaning

File: content/users-guide/cockpit-bundle/smart-rules.md [hunk 13]
```diff
@@ -16,7 +16,7 @@ helpcontent:
 
 To easily create rules, the Cockpit application includes a smart rules builder which allows you to create rules from templates (so-called smart rule templates).
 
->**Info:** Smart rules are only visible, if the tenant is subscribed to the Smartrule application. To manage smart rules, the user has to have CREATE permission for Inventory and either "Smartrule" permission or "CEP management" permission.
+>**Info:** Smart rules are only visible, if the tenant is subscribed to the Smartrule application. To manage smart rules, the user must CREATE permission for Inventory and either "Smartrule" permission or "CEP management" permission.
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/smart-rules.md:19]: must > must have

File: content/concepts/introduction-bundle/api.md (hunk 1 above)
Comment (JoshuaAdemola) [reply content/concepts/introduction-bundle/api.md:8]: Fixed

File: content/device-sdk/mqtt-bundle/device-certificates.md [hunk 14]
```diff
@@ -332,7 +332,7 @@ Upload your CA (or intermediate) certificate to the platform. This operation wil
 
 **Via UI:**
 
-1. You have to open the device management application, then navigate to the **Management** tab and select **Trusted certificates**.
+1. You must Open the Device Management application, then navigate to the **Management** tab and select **Trusted certificates**.
```
Comment (BeateRixen) [inline content/device-sdk/mqtt-bundle/device-certificates.md:335]: Remove "You must"

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]
